template.yaml

```diff
@@ -0,0 +1,96 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: >
+    NVA DynamoDB Resources stream reader trigger
+
+#Metadata:
+#  AWS::ServerlessRepo::Application:
+#    Name: NvaDynamoDBUpdateTrigger
+#    Description: NVA DynamoDB utility for reading streams and events
+#    Author: Unit
+#    SpdxLicenseId: MIT
+#    LicenseUrl: LICENSE
+#    Labels: ['${GIT_REPO}', '${CODEBUILD_RESOLVED_SOURCE_VERSION}', '@${BUILD_TIMESTAMP}']
+
+Globals:
+  Function:
+    Timeout: 20
```

Comment:
Optimistic?

Reply:
This is 20 seconds timeout - not SO very optimistic... But do we need i at all - API-gateway has 29 seconds timeout anyway...